diesel/src/query_dsl/mod.rs

```diff
@@ -875,8 +875,10 @@ impl<T: Table> QueryDsl for T {}
 pub trait RunQueryDsl<Conn>: Sized {
     /// Executes the given command, returning the number of rows affected.
     ///
-    /// Used in conjunction with [`insert_into`](../fn.insert_into.html),
-    /// [`update`](../fn.update.html) and [`delete`](../fn.delete.html)
+    /// `execute` is usually used in conjunction with [`insert_into`](../fn.insert_into.html),
+    /// [`update`](../fn.update.html) and [`delete`](../fn.delete.html) where the number of
+    /// affected rows is often enough information. When asking the database to return information
```

Comment:
Could you move `When asking...` on a new line?

Also what you say on `When asking the database to return data from query`.

- Information is also a number of the affected rows
- Operation feels to me a bit too much agnostic

Reply:
Hi @retep007, thanks for the review!

Can you take another look? I integrated your feedback here, and also tightened up the phrasing to be a little more prescriptive: "`load` should probably be used instead" instead of "`load` might be useful instead".